### PR TITLE
Translate some texts from Dashboard Page

### PR DESCRIPTION
**Pull request by berviantoleo**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [x] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description

Translate some texts from Dashboard Page

## Related Tickets & Documents

#14888

https://dev.to/devteam/forem-hacktoberfest-let-s-internationalize-404n

## Added/updated tests?

- [ ] Yes
- [x] No, and this is why: existing regression tests should suffice
- [ ] I need help with writing tests

## [optional] What gif best describes this PR or how it makes you feel?

Excited GIF

### CODE REVIEW

**Comment by berviantoleo** [comment]
Need help for french translation.

File: config/locales/fr.yml [hunk 1]
```diff
@@ -30,10 +33,16 @@ fr:
...
     see_all: Voir tout
+    series: Séries
     trending_on: Populaire sur
     view: Vue
     week: Semain
     year: Anée
+  dashboard:
+    empty_follower: Vous n'avez pas encore d'abonnés...
+    empty_post: C'est ici que vous pouvez gérer vos messages, mais vous n'avez encore rien écrit
+    upload_a_video: Upload a video
```
Comment (metamoni) [inline config/locales/fr.yml:44]: This should be `Charger une vidéo` or `Télécharger une vidéo`

Comment (berviantoleo) [reply config/locales/fr.yml]: Sure, thank you. 👍

**Comment by msarit** [comment]
Hey @berviantoleo! 👋🏾 
Thanks for this PR! Would you please merge `main` branch into yours and fix the conflicts? 🙏🏾

**Comment by berviantoleo** [comment]
> 
> 
> Hey @berviantoleo! 👋🏾 Thanks for this PR! Would you please merge `main` branch into yours and fix the conflicts? 🙏🏾

Sure. I've updated my branch and resolve the conlicts.

**Comment by msarit** [comment]
Hey @berviantoleo I'm reviewing your PR. 👋🏾 

I noticed that you named your feature branch `main`. Per our [Contributing Guidelines](https://developers.forem.com/contributing-guide/forem#how-to-contribute) (check item 2), we advise that you name your feature branch more descriptively, ideally a name that summarizes the issue being solved. 👍🏾 

Thank for contributing to Forem! 🙏🏾

**Comment by berviantoleo** [comment]
So, I need to recreate the PR and change the branch, right?

**Comment by berviantoleo** [comment]
Hi @msarit , I will close this PR and move to #14963 .

Thank you.

**Pull request closed without merging**